pages/blog/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { Layout } from '../../layouts/article/layout';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { load } from 'cheerio';
+import { Blog } from '../../components/blog';
+import { sortBy } from 'lodash';
```

Comment:
import this as `import sortBy from 'lodash/sortBy` to reduce the package size. I think otherwise we might get the whole lodash to our bundle.

also if axios isnt 100% needed I think fetch could reduce bundle size a bit too